cmd/argo/commands/cron/update.go

```diff
@@ -0,0 +1,105 @@
+package cron
+
+import (
+	"context"
+	"fmt"
+
+	"log"
+	"os"
+
+	"github.com/argoproj/pkg/errors"
+
+	"github.com/spf13/cobra"
+
+	"github.com/argoproj/argo-workflows/v3/cmd/argo/commands/client"
+	cronworkflowpkg "github.com/argoproj/argo-workflows/v3/pkg/apiclient/cronworkflow"
+	wfv1 "github.com/argoproj/argo-workflows/v3/pkg/apis/workflow/v1alpha1"
+	"github.com/argoproj/argo-workflows/v3/workflow/util"
+)
+
+type cliUpdateOpts struct {
+	output string // --output
+	strict bool   // --strict
+}
+
+func NewUpdateCommand() *cobra.Command {
+	var (
+		cliUpdateOpts  cliUpdateOpts
+		submitOpts     wfv1.SubmitOpts
+		parametersFile string
+	)
+	command := &cobra.Command{
+		Use:   "update FILE1 FILE2...",
+		Short: "update a cron workflow",
+		Run: func(cmd *cobra.Command, args []string) {
+			if len(args) == 0 {
+				cmd.HelpFunc()(cmd, args)
+				os.Exit(1)
+			}
+
+			if parametersFile != "" {
+				err := util.ReadParametersFile(parametersFile, &submitOpts)
+				errors.CheckError(err)
+			}
+
+			updateCronWorkflows(cmd.Context(), args, &cliUpdateOpts, &submitOpts)
+		},
+	}
+
+	util.PopulateSubmitOpts(command, &submitOpts, &parametersFile, false)
+	command.Flags().StringVarP(&cliUpdateOpts.output, "output", "o", "", "Output format. One of: name|json|yaml|wide")
+	command.Flags().BoolVar(&cliUpdateOpts.strict, "strict", true, "perform strict workflow validation")
+	return command
+}
+
+func updateCronWorkflows(ctx context.Context, filePaths []string, cliOpts *cliUpdateOpts, submitOpts *wfv1.SubmitOpts) {
+	ctx, apiClient := client.NewAPIClient(ctx)
+	serviceClient, err := apiClient.NewCronWorkflowServiceClient()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	fileContents, err := util.ReadManifest(filePaths...)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	var cronWorkflows []wfv1.CronWorkflow
+	for _, body := range fileContents {
+		cronWfs := unmarshalCronWorkflows(body, cliOpts.strict)
+		cronWorkflows = append(cronWorkflows, cronWfs...)
+	}
+
+	if len(cronWorkflows) == 0 {
+		log.Println("No CronWorkflows found in given files")
+		os.Exit(1)
+	}
+
+	for _, cronWf := range cronWorkflows {
+
```

Comment:
```suggestion
```

nit: unnecessary new line.

this unnecessary new line seems to exist in `argo cron create` as well (and may have been copied from there) and should be removed there too

Reply:
deleted it.